src/psl.c

```diff
@@ -78,12 +78,7 @@ typedef SSIZE_T ssize_t;
 
 #ifndef _WIN32
 # include <arpa/inet.h>
-#endif
-
-#ifdef HAVE_ALLOCA_H
-#	include <alloca.h>
-#elif defined _WIN32
-#	include <malloc.h>
+# include <malloc.h>
```

Comment:
I assume you mean stdlib.h here? This is a glibc-ism.

Reply:
Regarding https://learn.microsoft.com/de-de/cpp/c-runtime-library/reference/malloc?view=msvc-170, `malloc()` requires `stdlib.h` and `malloc.h`. The first is included further up, unconditionally.

Glibc only requires `stdlib.h`.

Reply:
Right but for non Windows the canonical definition is stdlib.h -- malloc.h may not exist and isn't documented as the correct way to do it.

It does exist on glibc but it doesn't exist on macOS, which caused a build failure.

Reply:
My bad, the include was in the `#ifndef _WIN32` path, should have be in the `#else` path. Fixed it and force-pushed.

Reply:
@eli-schwartz Can you check again (see my comment https://github.com/rockdaboot/libpsl/pull/231#discussion_r1481061750) and unblock the PR if OK?

Reply:
Yup, sorry, lost track of that.